feathr_project/test/test_feature_anchor.py

```diff
@@ -140,12 +140,12 @@ def test_agg_anchor_to_config():
     expected_agg_feature_config = """
             aggregationFeatures: {
                 source: nycTaxiBatchSource
-                key: [DOLocationID]
+                key.sqlExpr: [DOLocationID]
                 features: {
                     f_location_avg_fare: {
                         def: "cast_float(fare_amount)"
                         window: 90d
-                        agg: AVG
+                        aggregation: AVG
```

Comment:
What does this change do?

Reply:
It should be aggregation instead of agg.